First draft of ANS abstract (#66)
arfc/pride: publications/abstracts/reservoir-computing/methodology.tex

\section{Methodology}

> This work consists of three parts, data generation model training, and

Reviewer: ```suggestion
This work consists of three parts: data generation, model training, and
```

prediction. To generate

> data needed for training our RC model we used the \texttt{TrainARMA}

Reviewer: ```suggestion
data needed for training a reservoir computing model we used the \texttt{TrainARMA}
```

functionality
in the \texttt{RAVEN} tool from INL to generate synthetic histories. These
scenarios

> are passed to the RC model for training and testing.

Reviewer: ```suggestion
are passed to the reservoir computing model for training and testing.
```

\subsection{Training Data Generation}
In this work we use the scenario generation method described in Baker et. al
using the \texttt{RAVEN} framework \cite{baker_optimal_2018}. The algorithm for
producing synthetic histories is:
\begin{enumerate}
	\item Create a ``typical year`` from historical data.
	\item Fit an Autoregressive Moving Average (ARMA) model using
	\texttt{TrainARMA}.
	\item Use Monte Carlo sampling to generate synthetic histories from ARMA
	 model.
\end{enumerate}

> We perform this process for three datasets: Electricity demand, wind generation,

Reviewer: ```suggestion
We perform this process for three datasets: electricity demand, wind generation,
```

> and solar generation. This allows us to train an RC model to predict generation

Reviewer: ```suggestion
and solar generation. This allows us to train a reservoir computing model to predict generation
```

from wind and solar farms and to predict the net demand profile given by
\begin{equation}
	\label{eqn:net-demand}
	\begin{split}
		D_{net} & = (D_{total} - P_{wind} - P_{pv})_h \text{ $\forall$ $h$ in }

> 		[0,8759]

Reviewer: ```suggestion
		[0,8759].
```

	\end{split}
\end{equation}

Where $D_{total}$ is the total demand at a given hour, $h$, of a synthetic
demand profile from the
ARMA model.

\subsection{Reservoir Computing Model}

> Unlike a typical feed forward neural network, a reservoir computer (sometimes

Reviewer: I think that, in order to reference a "feed forward neural network" as if it's typical, you'll need to introduce it to the reader.

> called an ``echo state network'' or a ``liquid state machine'') is a type of

Reviewer: Citation for these alernative names?
